### ISSUE TITLE
Upload a pretrained model

### ISSUE THREAD

**Issue by tyagi-iiitv** [tyagi-iiitv/PointPillars#8]
We can upload a saved model.h5 file after training for around 150+ epochs for a direct usage when someone wants to try out this model on their data.

**Comment by zq07075335** [comment]
can you upload a example codeof h5 to pb  or model.pb ? I convert the h5 to pb，but the results of prediction were different. thx

**Comment by tyagi-iiitv** [comment]
That's actually a good point, we can directly save the model to .pb format instead of the older .h5 format. The code to be modified should be inside the point_pillars_training_run.py file. I'll update this issue once I finish working on it. In case you are able to make it work, please send a PR. Thanks!

**Comment by omarhassan91** [comment]
How did you convert the .h5 to a .pb model? I tried using pillar_net.save('point_pillars') but that resulted in an unfrozen .pb model that I couldn't figure out how to freeze correctly...

Thanks!

**Comment by tyagi-iiitv** [comment]
We never actually changed the code to save the model as a .pb file. As of now, the model is still saved as a .h5 file which can be loaded directly (for example in the point_pillars_prediction.py file). Also, the model is automatically saved based on the best loss value in the training script, so I'm not sure why would you need to save the model separately. Do you have any specific scenario here?

**Comment by omarhassan91** [comment]
I'm just trying to extract the .pb model to see if it's possible to do any sort of optimizations with it...are the steps that I followed with adding 'pillar_net.save('point_pillars') wrong to get these files (checkpoint, .index, .meta and .data)?

**Comment by tyagi-iiitv** [comment]
I updated the readme with the instructions. Closing this for now.
